lib/configs/browser.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     'github/no-blur': 'error',
     'github/no-dataset': 'error',
     'github/no-innerText': 'error',
-    'github/unescaped-html-literal': 'error'
+    'github/unescaped-html-literal': 'error',
+    'github/no-useless-passive': 'error',
+    'github/no-non-passive-high-frequency-events': 'error',
```

Comment:
Can we replace the double negative with a positive assertion: require-passive-events?